pkg/etcd/pki.go

```diff
@@ -63,7 +63,7 @@ func (p *etcdProcess) createKeypairs(peersCA *pki.Keypair, clientsCA *pki.Keypai
 
 		certConfig := certutil.Config{
 			CommonName: me.Name,
-			Usages:     []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
+			Usages:     []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth},
```

Comment:
Nit: A comment with a link to e.g. https://github.com/etcd-io/etcd/issues/9785 would be helpful for people scratching their head about this one in future :-)

Reply:
added a comment with a link to the original issue